index.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module "svelte-virtual-list" {
```

Comment:
The code shouldn't be wrapped inside a `declare module`, like this TS won't pick it up correctly